apps/freelance/src/pages/Freelancer/CreateProfile1/index.tsx

```diff
@@ -0,0 +1,135 @@
+import { useTranslation } from "react-i18next";
+import { StyledPage, Form } from "./style";
+import {
+	ThemeColors,
+	ThemeBackground,
+	Dashboard,
+	StyledTitle,
+	StyledButton,
+} from "@freelance/components";
+import { ThemeProvider } from "styled-components";
+import countries from "utils/select-options/eu-countries.json";
+import categories from "utils/select-options/categories.json";
+import hourRate from "utils/select-options/hour-rate.json";
+import workHours from "utils/select-options/hours-amount.json";
+import skills from "utils/select-options/skills.json";
+import workExperience from "utils/select-options/work-experience.json";
+import employmentType from "utils/select-options/employment-type.json";
+import englishLevel from "utils/select-options/english-level.json";
+
+export function CreateProfile1() {
+	const { t } = useTranslation();
+
+	return (
+		<ThemeProvider theme={ThemeColors && ThemeBackground}>
+			<StyledPage>
+				<Dashboard userRole="freelancer">
+					<StyledTitle tag="h2" fontSize="md" fontWeight={700}>
+						{t("dashboard.profilePage.title")}
+					</StyledTitle>
+					<Form>
+						<input
+							id="fullName"
+							type="text"
+							name="fullName"
+							placeholder={t("freelancer.createProfile.fullNamePlaceholder")}
+							required
+						/>
+						<div className="selectContainer">
+							<select id="country" name="country" required>
+								<option value="" disabled selected hidden>
+									{t("freelancer.createProfile.selectOption.country")}
+								</option>
+								{countries.map(({ name }) => {
```

Comment:
will you always have countries list?

Reply:
Yes, this list will always be shown to the user to select one option from the list.
According to your comment on slack, I should merge if the answer is no. But I am not sure what should I do, since it is yes. Could you point some directions on what should I be doing, please?